### DISCUSSION TITLE
Playlists - multiple folders - cant get it to work

### DISCUSSION THREAD

**Discussion by M0M0M3** [gerbera/gerbera#3051] [General]
So my user case for Gerbera is to have it be an Upnp server for my music files, which I then can access with Cambridge Audio CXN100. I can see all my music files and play them, but I wanted to know if there is a way to create playlists that work with multiple folders.

When I have a playlist located in the Album directory, playing only that album, then it works fine. Playlist is m3u file and looks something like this:

```
#EXTM3U
#EXTINF:996,Brian Eno - 1/1
A1 - 1-1.flac
```

When I tired to create a playlist with multiple folders then I have not gotten it to work. After creating playlist I edited the file path to point to the mounted path for the docker container, but cannot get it to play. An example below with the file path.

```
#EXTM3U
#EXTINF:215,Montañera - A Flor de Piel
/mnt/content/SMusic2024/A Flor de Piel - Montañera (2023)/01 - A Flor de Piel.flac
```

The music in docker container is set up like below, and then this contains a bunch of subfolders:

```
    volumes:
      - /srv/mergerfs/Hydra/media/MUSIC:/mnt/content:ro
```

Any help or advice is much appreciated.

**Comment by KarlStraussberger** [comment]
The key to your problem are the special spanish characters. Make sure your playlist ist utf8 as well as your file system.
You should check your logs and if still in trouble, run gerbera with `--debug` command line option to see more messages.

**Reply by M0M0M3** [reply]
Hey, thanks for your reply again! I tried again with playlist that has album with no special characters. Still no luck unfortunately. 

I haven't been able to get extra debugging working with docker compose. so have to look into this some more. 

I also have not yet built a docker container from scratch,  without using docker compose, so feeling a bit hesitant to do that. But will also look into this.

**Reply by KarlStraussberger** [reply]
I recently created playlists myself and it worked like a charm. I only experienced that after saving the file was not reimported until the next restart.
